web_external/Literature/publications.bibtex

```diff
@@ -1,9 +1,9 @@
-@article{yuan2017automatic,
-  title={Automatic skin lesion segmentation with fully convolutional-deconvolutional networks},
-  author={Yuan, Yading and Chao, Ming and Lo, Yeh-Chi},
-  journal={arXiv preprint arXiv:1703.05165},
-  year={2017},
-  url={https://arxiv.org/pdf/1703.05165.pdf}
+﻿@Article{yuan2017,

```

Comment:
Entry types are case sensitive, and all of the [known entry types](https://en.wikipedia.org/wiki/BibTeX#Entry_types) are all-lowercase.

So, for example, this should be `@article`.